website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -232,6 +234,25 @@ DimensionType = [CATEGORICAL, TIME]
 }
 ```
 
+#### List a saved query
+
+List a single saved query using environment ID and query name.
+
+```graphql
+
+{
+    savedQuery(environmentId: $environment_id, savedQueryName: "saved_query_name") {
+        name
+        queryParams {
```

Comment:
Can we add all the same fields as the `savedQueries` here too?

Reply:
Added in [this commit](https://github.com/dbt-labs/docs.getdbt.com/pull/7430/commits/86b6dc84016efb3c4488605a3053b949099c49b9)